doc/source/graphicsItems/make

```diff
@@ -11,6 +11,7 @@ GraphicsWidget
 GridItem
 HistogramLUTItem
 ImageItem
+PColorMeshItem
```

Comment:
I'm not actually sure these "make" files do anything in the current docs build. I'm not aware of something like this being part of sphinx - my guess is this was part of an older build setup to generate the stub files for the API reference. Could you copy one of the other files in the `doc/source/graphicsItems` folder?

Reply:
I do not understand what you mean by "Could you copy one of the other files in the `doc/source/graphicsItems` folder?".

Reply:
For now at least, the API docs are manually created, so the .rst files in `doc/source/graphicsItems/` are all stub files with pretty much the same content. So the idea was to copy one of those files and replace the class name with `PColorMeshItem`. legenditem.rst would be a good example since it includes the top class-level doc and the `__init__` method.

Reply:
Done, the file looks like this:
```rst
PColorMeshItem
==============

.. autoclass:: pyqtgraph.PColorMeshItem
    :members:

    .. automethod:: pyqtgraph.PColorMeshItem.__init__

```